React/Base/RCTRootContentView.m

```diff
@@ -86,6 +86,14 @@ - (void)updateAvailableSize
     return;
   }
 
+  //unnecessary to update avaliableSize, it's triggered by setBounds: from RCTUIManager
+  if (self.superview) {
+    CGRect superviewFrame = self.superview.bounds;
+    if (!CGRectEqualToRect(superviewFrame, self.frame)) {
```

Comment:
Okay, but what is an idea/reasoning behind this changeset?
Comparing `view.frame` with `superview.bounds` makes no any sense, I am affraid.

Reply:
I always think that the size of content view should be equal to its super view's size. That's why you set content's frame in the function layoutSubviews of rootView ? 
```
- (void)layoutSubviews
{
  [super layoutSubviews];
  _contentView.frame = self.bounds;  // here
  _loadingView.center = (CGPoint){
    CGRectGetMidX(self.bounds),
    CGRectGetMidY(self.bounds)
  };
}
```

if `view.frame`  != `superview.bounds`, it's invalidate

Reply:
@shergin what's your  opinion?